platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/uptime/UptimeTracker.java

```diff
@@ -62,6 +63,11 @@ public class UptimeTracker {
      */
     private final StatusActionSubmitter statusActionSubmitter;
 
+    /**
+     * The uptime data for all the nodes.
+     */
+    final UptimeData uptimeData;
```

Comment:
member should be `private`

Reply:
It's package-private because it's used in the test. Would you prefer package-private getter?

Reply:
I added a comment to javadoc about it